test/e2e/volumes.go

```diff
@@ -381,9 +381,8 @@ func deleteCinderVolume(name string) error {
 	return err
 }
 
-// These tests need privileged containers, which are disabled by default.  Run
-// the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
-var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
+// These tests need privileged containers, which are disabled by default.
```

Comment:
I changed the common func a while ago to allow setting privileged to false.  So maybe as a future pr, it can be investigated if these pods can be run as unprivileged.